### DISCUSSION TITLE
Latest generation lens supported + tutorial

### DISCUSSION THREAD

**Discussion by Jbla0712** [ptrumpis/snap-camera-server#115] [General]
Hello, still no solution for recent lenses like "my new twin" or "sibling".
I tried to patch them with the tool but without success.
I tried with "my new twin" 60928810985 to deactivate fallback mode. Without success

Edit: 👇The location of the lens is good, 
you can follow the example to add your modified lenses in your folder snap-camera-server/import

[
![snap](https://github.com/ptrumpis/snap-camera-server/assets/13618418/bfa0d74d-05fc-4239-be84-5567bd87ee21)
](url)

**Comment by Jbla0712** [comment]
Sorry, I didn't understand the missing assets and private directory!
All lenses work very well after fix with snap tools. 
I confirm that the "Silbing" or "my new twin" lens works really well afetr fix fallback mode (sometime blendmode), for example.

**I think you should make a new, clearer tutorial**

Pat is just the boss 
Thanks (No problem supporting your work every month )

Love

**Reply by Orpheusll** [reply]
Hey dude, I really want to figure out how to do this for newer lenses, could you by any chance help me out? I can aid you through patreon support or buymeacoffee too for the help

**Reply by Jbla0712** [reply]
Hi friend. Rather support PTrumpis 🧙‍♀️
This is his work and I thank him for leaving all of this open source.

I'm going to do a tutorial that starts from the beginning. With an AI voice in English. (I am not native English). A little bit of work.
Patrick (ptrumpis) will release a more complete tool for us in the future.
It's not simple, yes. The tutorials and instructions go in all directions. Yet there is nothing difficult once understood.


It's normal, it's still a draft, but Pat gave all the tools for advanced developers (I'm not a developer at all, I'm self-taught).
Very complete but not novice friendly it is true.
In short, I will try to help you

A self-taught person (not a developer) will perhaps make a better tutorial for the general public. I think it makes sense. 
On the other hand, I'm going to shit about the editing 😂😒
 (not for the voice, but the video 😁)

**Comment by Jbla0712** [comment]
For the video it's not bad for a first draft. I'm really bad at video editing.👨‍🏭

I used Elevenlab for the first voice and Bark for the second. (I usually use RVC from Github). It was easier online.
SO,
Tell me if this is understandable. 🙈🐒
There are some rough passages due to video cut (non-chronological rush video). But I think it will help you though.

VIDEO TUTORIAL
https://youtu.be/Pp6ljLzRjnU?si=9uk7DCv_GbI9r10W


 💥For a lot of lenses you don't need to change the blendmode. 
You just have to deactivate fallback mode with the tools.
  ⚡⚡Try deactivating fallback mode before touching blendmode and dispack the lens ⚡⚡
I didn't do it in the video.
👉For sur you can try different blendmode fore advended non working lens, but usually "blendmode=13" resolve the most of lens.
Have fun
